documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -5,7 +5,9 @@
 ### Changed features
 
 * Support for Dart is now extended to Dart 3.1.2 and Flutter 3.13.4.
+* [solution_name] will now wait for a number of seconds specified by [blackduck_product_name] before attempting to retry creating a new scan when [blackduck_product_name] is busy.
```

Comment:
Tweak suggestion, just reversing the statement: 
When [blackduck_product_name] is busy, [solution_name] will now wait the number of seconds specified by [blackduck_product_name] before attempting to retry scan creation.